### ISSUE TITLE
Build static files

### ISSUE THREAD

**Issue by arttemiuss43** [gridsome/gridsome#72]
Every time I build static files with gridsome it compiles assets. I wonder if It's possible to rebuild static files only from GraphQL schema on production (when using webhooks to update data) and not compile assets over and over? It would reduce build time drastically.

**Comment by hjvedvik** [comment]
I agree :) We will try to make this possible before a `1.0` release. The `static-query` is generated by webpack now, but I want to find a solution where they can be generated together with `page-query` in order to skip the compilation step.

**Comment by arttemiuss43** [comment]
It also would be very useful if gridsome cli had `--no-build` option to update pages without rebuilding the entire application. So we could skip the compilation process only when we need it.
